packages/devtools/lib/src/ui/html_icon_renderer.dart

```diff
@@ -314,8 +314,8 @@ class _MaterialIconRenderer extends HtmlIconRenderer<MaterialIcon> {
   @override
   CanvasImageSource createCanvasSource() {
     final canvas = CanvasElement(
-      width: iconWidth * window.devicePixelRatio,
-      height: iconHeight * window.devicePixelRatio,
+      width: (iconWidth * window.devicePixelRatio).toInt(),
```

Comment:
I don't have familiarity with this code, so don't know if a trunc or a round is best here; @jacob314 .

Reply:
Rounding to the nearest pixel is probably best.

Reply:
Go ahead and revert this file from the CL. Turns out getting this case exactly right was a bit more complicated to deal with windows 10 devices with floating point dpis correctly.
https://github.com/flutter/devtools/pull/376